src/components/TransactionsList.js

```diff
@@ -1,6 +1,19 @@
 import React from 'react'
+import Transaction from './Transaction'
 
-const TransactionsList = () => {
+const TransactionsList = (props) => {
+  const origList = props.transactions
+  const filteredList = origList.filter(transaction => {
```

Comment:
This may be more readable if you can split this up into different helper methods. You might even consider making this a class based component. 

The other way to do this would be to do the filtering on the Container, that way you can add some methods there.